### ISSUE TITLE
fix error where _merge already exists

### ISSUE THREAD

**Issue by luispfonseca** [luispfonseca/stata-rcallcountrycode#4]
``` stata
If no error happened, then it likely found either 0 matches or more than 1 for these cases
Merging the data
variable _merge already defined
r(110); t=2.88 19:17:47
```

**Comment by luispfonseca** [comment]
fixed in 5e0e0a3 by creating tempvar to store merge results

**Comment by echo-charbel** [comment]
After each merge, you need to drop the variable "_merge"
